src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/Ordering/PasteOrdering.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands.Ordering
+{
+    /// <summary>
+    /// This does proper file ordering for pasting or dropping items into a folder or project.
+    /// </summary>
+    [Export(typeof(IPasteDataObjectProcessor))]
+    [Export(typeof(IPasteHandler))]
+    [AppliesTo(ProjectCapabilities.SortByDisplayOrder)]
+    [Order(OrderPrecedence)]
```

Comment:
use Order.Default.

Reply:
I can't use Order.Default because it is not enough precedence.